src/main/java/com/carmen/app/boundary/CarResource.java

```diff
@@ -35,15 +40,31 @@
 	private CarService carService = new CarService();
 
 	@GET
+	@Operation(summary = "Get all cars",
```

Comment:
The annotations are correct but as you can see, we need to put a lot of annotation that is not really relevant for the application logic. To keep this class clean, please create an interface and add the swagger annotation to that interface, this way we keep the Resource class cleaner. 😄